constellation-app/constellation
# Plugin Reporter Null Pointer on Drop-Down List

### Description of the Change
The filter drop-down list on the Plugin Reporter often causes a null pointer exception in a variety of situations. This change spots the null pointer from occurring. 


### Alternate Designs
N/A


### Why Should This Be In Core?
Stops a null pointer from occurring. 


### Possible Drawbacks
None realised


### Verification Process
There were at least 2 ways of replicating the null pointer exception. Both were tested to ensure it no longer occurred.

1st:
1. Open a new graph
2. Open Plugin Reporter and click on the drop down list of filters
3. Create a Sphere Graph
4. Make 2 selections on the graph (This step would throw the null pointer)

2nd:
1. Open the Plugin Reporter
2. Build a new sphere graph
3. Click on the drop down list
4. Open a new graph
5. Close the new graph (This step would throw the null pointer)




### Applicable Issues
#930
